tests/test_sketch.py

```diff
@@ -792,3 +792,16 @@ def test_polydata_methods():
     assert len(pd) == 2
     assert len(pd_edges) == 1
     assert len(pd_faces) == 1
+
+
+def test_sketch_pyconus2024_voglster_issue1195():
+    """Test sketching unexpected behaviour observed in PyConUS 2024 by @voglster."""
```

Comment:
```suggestion
    """Test sketching unexpected behavior observed in PyConUS 2024 by @voglster."""
```
In general, we always want to use U.S. spellings--however, I'm not sure if this quoting something copyrighted?

Reply:
All good @PipKat - let me add your suggestion!